Client/Frontend/Widgets/Menu/Items/MenuItemView.swift

```diff
@@ -36,7 +36,6 @@ class MenuItemCollectionViewCell: UICollectionViewCell {
         menuTitleLabel.snp_makeConstraints { make in
             make.top.equalTo(menuImageView.snp_bottom).offset(padding)
             make.centerX.equalTo(self)
-            make.bottom.greaterThanOrEqualTo(self).offset(-padding)
             make.left.lessThanOrEqualTo(self).offset(2 * padding)
```

Comment:
The menu cell just grows and grows to fit the text. This was resolved in a previous bug and is what highlighted the need for this issue